bin/blackbox_less

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+#
+# blackbox_less -- Decrypt a file, less it, shred it
+#
+set -e
+source "${0%/*}/_blackbox_common.sh"
+
+for param in "$@" ; do
+  shreddable=0
+  unencrypted_file=$(get_unencrypted_filename "$param")
+  if [[ ! -e "$unencrypted_file" ]]; then
+    "${BLACKBOX_HOME}/blackbox_edit_start" "$param"
+    shreddable=1
+  fi
+  less "$unencrypted_file"
```

Comment:
Rather than `less`, use:

    ${PAGER:-less}

That way people can use any pager they want.  If $PAGER is not set, less will be used.